### PR TITLE
Automated Prometheus version update 0.53.1-30.1.0

### PR DESCRIPTION
**Pull request by replicated-ci-kurl**
Automated changes by the [cron-prometheus-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-prometheus.yaml) GitHub action

### CODE REVIEW

**Comment by DanStough** [comment]
DON'T MERGE. Problem with upstream https://github.com/prometheus-community/helm-charts/issues/1500

File: addons/prometheus/template/base/install.sh [hunk 1]
```diff
@@ -14,7 +15,9 @@ function prometheus() {
 
     grafana_admin_secret "$src" "$operatordst"
 
-    kubectl apply -k "$crdsdst/"
+    # Server-side apply is needed here because the CRDs are too large to keep in metadata
+    # https://github.com/prometheus-community/helm-charts/issues/1500
+    kubectl apply -k --server-side "$crdsdst/"
```
Comment (DanStough) [inline addons/prometheus/template/base/install.sh:20]: These are the changes I made to the template. Everything else is generated.

File: addons/prometheus/template/base/install.sh [hunk 2]
```diff
@@ -14,7 +15,9 @@ function prometheus() {
 
     grafana_admin_secret "$src" "$operatordst"
 
-    kubectl apply -k "$crdsdst/"
+    # Server-side apply is needed here because the CRDs are too large to keep in metadata
+    # https://github.com/prometheus-community/helm-charts/issues/1500
+    kubectl apply --server-side -k "$crdsdst/"
```
Comment (DanStough) [inline addons/prometheus/template/base/install.sh:20]: These are the changes I made to the template. Everything else is generated.

File: addons/prometheus/template/base/install.sh [hunk 3]
```diff
@@ -14,7 +15,9 @@ function prometheus() {
 
     grafana_admin_secret "$src" "$operatordst"
 
-    kubectl apply -k "$crdsdst/"
+    # Server-side apply is needed here because the CRDs are too large to keep in metadata
+    # https://github.com/prometheus-community/helm-charts/issues/1500
+    kubectl apply --server-side --force-conflicts -k "$crdsdst/"
```
Comment (DanStough) [inline addons/prometheus/template/base/install.sh:20]: These are the changes I made to the template. Everything else is generated.

**Review by divolgin — APPROVED** [review]

**Pull request merged**